packages/sdk-rtl/src/transport.ts

```diff
@@ -379,8 +380,12 @@ export type Values = { [key: string]: any } | null | undefined
 export function encodeParam(value: any) {
   if (value instanceof Date) {
     value = value.toISOString()
+  } else if (value instanceof DelimArray) {
```

Comment:
This looks so familiar. I thought we'd fixed it

Reply:
at least we've fixed it now